api/queries_repo.go

```diff
@@ -500,13 +500,16 @@ type repositoryV3 struct {
 }
 
 // ForkRepo forks the repository on GitHub and returns the new repository
-func ForkRepo(client *Client, repo ghrepo.Interface, org string) (*Repository, error) {
+func ForkRepo(client *Client, repo ghrepo.Interface, org, newName string) (*Repository, error) {
 	path := fmt.Sprintf("repos/%s/forks", ghrepo.FullName(repo))
 
 	params := map[string]interface{}{}
 	if org != "" {
 		params["organization"] = org
 	}
+	if newName != "" {
+		params["name"] = newName
```

Comment:
Do we need to worry about older GHES versions having failure responses here because of unknown parameters?

Reply:
That's a good question, but most (or all?) REST API endpoints will just ignore unknown params. I've tested with a GHE version that doesn't support this and the parameter is just ignored.